package.json

```diff
@@ -266,18 +266,20 @@
     ]
   },
   "scripts": {
-    "build-storybook": "build-storybook -c src/storybook/ -o build/storybook",
+    "build-storybook": "env NODE_OPTIONS=--openssl-legacy-provider build-storybook -c src/storybook/ -o build/storybook",
     "build-types": "graphql-codegen",
-    "build": "npm run build-types && webpack -p",
+    "prebuild": "npm run build-types",
+    "build": "env NODE_OPTIONS=--openssl-legacy-provider webpack -p",
     "check-strict-null-errors": "tsc --noEmit --strictNullChecks | node scripts/count-strict-null-check-errors.js",
     "check-types": "tsc --noEmit",
     "extract-json-messages": "formatjs extract 'src/**/*.{ts,tsx}' --out-file locale/defaultMessages.json --format scripts/formatter.js",
     "extract-messages": "npm run extract-json-messages",
     "fetch-schema": "graphql-codegen --config ./fetch-schema.yml && node scripts/build-schema.js",
     "heroku-postbuild": "npm run build",
     "serve:lhci": "NODE_ENV=production npm run server",
-    "start": "npm run build-types && webpack-dev-server -d",
-    "storybook": "start-storybook -p 3000 -c src/storybook/",
+    "prestart": "npm run build-types",
+    "start": "env NODE_OPTIONS=--openssl-legacy-provider webpack-dev-server -d",
```

Comment:
is that webpack thing? Please create an issue for that and link it in this PR

Reply:
Done in https://github.com/saleor/saleor-dashboard/issues/2557